test/new_relic/agent/database_test.rb

```diff
@@ -23,7 +23,7 @@ def test_adapter_from_config_symbol
   end
 
   def test_adapter_from_config_uri_jdbc_postgresql
-    config = {:uri => "jdbc:postgresql://host/database?user=posgres"}
+    config = {:uri => "jdbc:postgresql://host/database?user=postgres"}
```

Comment:
??

Reply:
Huh. Should be fine. We'll see failures in our JRuby tests if not.